### PR TITLE
fix 404 error when refreshing create webhook page

### PR DESCRIPTION
**Pull request by carlos-logro**
issue: #192 

# Changes
The approach I did was basically to search & remove the keyword `create` that was is not supposed to be present in the api uri call.

### CODE REVIEW

File: webhooks-extension/src/api/index.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ export function getSelectedRows(selectedRows) {
 
 export function getAPIRoot() {
   const { href, hash } = window.location;
-  let newHash = hash.replace('#/extensions','v1/extensions')
+  let newHash = hash.replace('#/extensions','v1/extensions');
+  if(hash.search('create') !== -1){
```
Comment (AlanGreene) [inline webhooks-extension/src/api/index.js]: with this approach you'll have to update this for other potential paths in future
why not just replace the whole hash with what you're expecting?

Comment (carlos-logro) [reply webhooks-extension/src/api/index.js]: soo basically hard code the newHash variable?

**Comment by carlos-logro** [comment]
@AlanGreene this PR was updated, i hardcoded the go to hash

**Pull request merged**